packages/firestore/test/unit/local/local_store.test.ts

```diff
@@ -643,22 +639,25 @@ function genericLocalStoreTests(
   });
 
   it('handles SetMutation -> Ack -> PatchMutation -> Reject', () => {
```

Comment:
This test could be moved to the `LocalStore w/ Memory Persistence` "subclass".

Reply:
I started implementing this, but I think it gets a little too ugly with sharing the test infrastructure with the generic tests. 

If you feel strongly we can refactor, but I'd like to do it in a separate PR.

Reply:
I was hoping that you could just move the tests method to the respective class, similar to what we do here: https://github.com/firebase/firebase-js-sdk/blob/322357244dfc3f35ae97e7d522d40d345b216b51/packages/firestore/test/unit/local/remote_document_cache.test.ts#L168

This can of course be done in a separate PR.

Reply:
I filed b/117272361 to do it across all clients